File: parcial/src/main/java/co/unicauca/parcial/dto/TelephoneDTO.java
package co.unicauca.parcial.dto;

import jakarta.validation.constraints.Pattern;
import lombok.*;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class TelephoneDTO {
    private Integer telephoneId;
    private String type;
    @Pattern( regexp = "[8][0-9]{5,20}", message = "{telephone.number}")
    private String number;
}

File: parcial-front/src/app/model/TelephoneDTO.ts
export class TelephoneDTO {
    telephoneId: number;
    type: string;
    number: string;

    constructor(telephoneId: number, type: string, number: string) {
        this.telephoneId = telephoneId;
        this.type = type;
        this.number = number;
    }
}
